cypress/integration/gift_card_select.spec.ts

```diff
@@ -0,0 +1,32 @@
+/// <reference types="Cypress" />
+import { HomePo } from '../support/home.po';
+import { GiftCardsPo } from '../support/giftCards.po';
+
+
+describe('Gift card choosing and adding to cart', () => {
+    const homePage = new HomePo();
+    const giftCardsPage = new GiftCardsPo();
+
+    beforeEach(() => cy.visit(homePage.pageUrl));
+
+    it('Gift card choosing and adding to cart check', () => {
+
+        const expectedPrice = '$100';
+        const expectedSuccessMsg = 'Added to Cart';
+
+        homePage.clickOnElemWithText(homePage.headerNavLinks, homePage.giftCardsLinkText);
+
+        giftCardsPage.isCurrentUrlInclude(giftCardsPage.pageUrl);
+
+        giftCardsPage.clickOnElem(giftCardsPage.giftCardPrintAtHome);
```

Comment:
Then user redirected to page with product route - this check not covered, should be smth like this:
cy.url().should('include', '/product/');